qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -82,6 +82,13 @@ class ExperimentStatus(enum.Enum):
     DONE = "experiment jobs and analysis have successfully run"
     ERROR = "experiment jobs or analysis incurred an error"
 
+    def __json_encode__(self):
+        return self.name
+
+    @classmethod
+    def __json_decode__(cls, value):
+        return cls.__members__[value]  # pylint: disable=unsubscriptable-object
```

Comment:
Why we cannot just `cls(value)` ?

Reply:
That doesn't work since value here is a string.